client/my-sites/promote-post/components/campaign-item/index.tsx

```diff
@@ -0,0 +1,75 @@
+import './style.scss';
+import { Button } from '@wordpress/components';
+import Badge from 'calypso/components/badge';
+import FoldableCard from 'calypso/components/foldable-card';
+import { Campaign } from 'calypso/data/promote-post/use-promote-post-campaigns-query';
+
+type Props = {
+	campaign: Campaign;
+};
+
+export default function CampaignItem( { campaign }: Props ) {
+	const header = (
+		<div className="campaign-item__header">
+			<div className="campaign-item__header-image">
+				<img src={ campaign.content_image } alt="" />
+			</div>
+			<div className="campaign-item__header-content">
+				<div className="campaign-item__header-title">{ campaign.content_config.title }</div>
+				<div className="campaign-item__header-status">
+					<Badge>todo { campaign.status_smart }</Badge>
+				</div>
+			</div>
+		</div>
+	);
+	return (
+		<FoldableCard header={ header }>
+			<div className="campaign-item__stats">
+				<div className="campaign-item__stats-row1">
+					<div className="campaign-item__reach">
+						<div className="campaign-item__reach-value">123 todo</div>
```

Comment:
sooner than later we will wire this to a new method to get the report of the campaign

Reply:
cool, I'll NOT touch this for now until we get more certainty.